cmd/influxd/upgrade/security.go

```diff
@@ -104,20 +104,20 @@ func upgradeUsers(ctx context.Context, v1 *influxDBv1, v2 *influxDBv2, targetOpt
 	return nil
 }
 
-// securityScriptHelper is a helper used by `generate-security-script` command.
-type securityScriptHelper struct {
+// securityUpgradeHelper is a helper used by `upgrade` command.
+type securityUpgradeHelper struct {
```

Comment:
The logic in this struct's methods still seemed to be useful, so I just renamed it to avoid future confusion